packages/flutter/lib/src/services/flutter_version.dart

```diff
@@ -15,7 +15,25 @@
 abstract final class FlutterVersion {
   const FlutterVersion._();
 
-  /// The Flutter version used to compile the app.
+  /// The value is a versioning string representing the version of Flutter
+  /// used to compile the app, possibly followed by whitespace and other
+  /// version and build details.
+  ///
+  /// Flutter uses a modified [CalVer](https://calver.org/) scheme.
+  /// The major version is incremented when the product team decides
+  /// there are features impactful enough to increment this value.
+  /// Minor is incremented on a monthly basis. Example: Flutter 3.0.0 shipped
+  /// in May 2022, meaning an August 2022 release would put the Flutter version
+  /// at 3.3.0 as it is 3 months after the last stable release.
+  /// The patch version is incremented whenever a hotfix is applied
+  /// to the current stable release.
```

Comment:
This looks accurate, so I don't object to adding it, but it feels redundant with Flutter's other documentation.

The same information is present at https://github.com/flutter/flutter/blob/master/docs/releases/Release-versioning.md , and it's better-formatted there and easier to read. That page is also more likely to stay up to date if details of this scheme change.

So my suggestion for this paragraph would be to replace it with a link to that page. If a wiki page feels too internal, I think a natural home for this information would be on the Flutter website, for example at https://docs.flutter.dev/release/release-notes . The website repo is https://github.com/flutter/website .